backend/Dockerfile.sessionspaces

```diff
@@ -9,21 +9,27 @@ RUN cargo install cargo-auditable
 COPY argo-workflows-openapi/Cargo.toml argo-workflows-openapi/Cargo.toml
 COPY graph-proxy/Cargo.toml graph-proxy/
 COPY sessionspaces/Cargo.toml sessionspaces/
+COPY telemetry/Cargo.toml telemetry/Cargo.toml
+COPY telemetry/build.rs telemetry/build.rs
+COPY telemetry/src/lib.rs telemetry/src/lib.rs
```

Comment:
Can we avoid copying in the telemetry source here and instead leave that until the final build?